### ISSUE TITLE
the division of data set

### ISSUE THREAD

**Issue by ljx97** [LTTM/SDR#7]
Hello,

Thank you for sharing the code. I'm interesting in your work. But I still have a question about  the division of the data set. How did you divide your test data set?

**Comment by umbertomichieli** [comment]
Hi, the split between training/val/test splits is the same as in the original training set.
About the division for the incremental training: we followed previous works on the topic, like [ILT](https://openaccess.thecvf.com/content_ICCVW_2019/html/TASK-CV/Michieli_Incremental_Learning_Techniques_for_Semantic_Segmentation_ICCVW_2019_paper.html) or [MiB](https://arxiv.org/abs/2002.00718).

To make it short: we consider in the test set all the images having only previously seen classes inside.
Hope that this helps.
Kind regards.
